index.js

```diff
@@ -144,18 +144,37 @@ const formatMarkdown = (text, images) => {
     return text;
 };
 
-export const askAI = async (message, useJSON = false) => {
-    if (useJSON) return await queryBard(message);
-    else return (await queryBard(message)).content;
+export const askAI = async (message, config) => {
+    let SNlM0e = undefined
+    let useJSON = false
+
+    if (typeof config == "string") {
+        SNlM0e = config
+    }else if (typeof config == "boolean") {
```

Comment:
So something like
```suggestion
    if (typeof config == "boolean") {
```

Reply:
Evan has came out with his idea; 
```
    let config = args.slice(0, 2)
    let SNlM0e = globalSNlM0e, useJSON = false;

    for (let property of config) {
        if (typeof property === "string") {
            SNlM0e = property;
        } else if (typeof property === "boolean") {
            useJSON = property;
        }
    }

```, I already implement this to my latest commit.